src/search.cpp

```diff
@@ -1018,7 +1019,7 @@ namespace {
                        && !pos.see_ge(make_move(to_sq(move), from_sq(move))))
                   r -= 2 * ONE_PLY;
 
-              ss->statScore =  thisThread->mainHistory[~pos.side_to_move()][from_to(move)]
+              ss->statScore =  thisThread->mainHistory[us][from_to(move)]
```

Comment:
Shouldn't this be ``` ~us ``` ?

Reply:
@mstembera No, because this part of the code is after do_move.